codes/kotlin/chapter_backtracking/n_queens.kt

```diff
@@ -51,9 +51,9 @@ fun backtrack(
 /* 求解 n 皇后 */
 fun nQueens(n: Int): List<List<List<String>>?> {
     // 初始化 n*n 大小的棋盘，其中 'Q' 代表皇后，'#' 代表空位
-    val state: MutableList<MutableList<String>> = ArrayList()
+    val state = ArrayList<MutableList<String>>()
```

Comment:
I'm confused about the `val state = ArrayList<MutableList<String>>()`, but `val copyState = ArrayList<List<String>>()`. Can you unify them?